.github/workflows/code.yml

```diff
@@ -171,8 +171,7 @@ jobs:
         uses: softprops/action-gh-release@1e07f4398721186383de40550babbdf2b84acfc5 # v0.1.14
         with:
           prerelease: ${{ contains(github.ref_name, 'a') || contains(github.ref_name, 'b') || contains(github.ref_name, 'rc') }}
-          files: |
-            dist/*
+          files: dist/*
```

Comment:
I assume you have verified that this works? I thought I had to add the /* to make it work previously - but perhaps we had subfolders then?